src/app/services/sensor.service.ts

```diff
@@ -110,32 +107,22 @@ export class SensorService {
 
   /** Register sensor */
   public register(sensor: IRegisterSensorBody) {
-    return this.http.post<ISensor>(`${environment.apiUrl}/Sensor`, sensor).toPromise();
+    return this.http.post<any>(`${environment.apiUrl}/Sensor`, sensor).toPromise();
```

Comment:
Waarom heb je hier any van gemaakt? Of komt er geen ISensor terug na de POST? Zou het mooi vinden als we zo veel mogelijk typing kunnen houden (en zo min mogelijk any) 😉

Reply:
Uit een register krijg je alleen de sensorId terug, zal daar even class voor aanmaken.